src/other-css/style.css

```diff
@@ -457,11 +459,6 @@ body:not(.mousedown) .extraselectedrelation:hover {
   visibility: hidden
 }
 
-.none {
```

Comment:
I think I use this class in some way, but I'm not sure.  See ui.js: 485 and ui.js:567

Reply:
I’ve removed it because it’s duplicated from [`src/sass/utility/_display.scss`](https://github.com/DCMLab/reductive_analysis_app/blob/ui/src/sass/utility/_display.scss#L1-L3)

Reply:
Ah gotcha!